modules/overview-of-upgrading-odh.adoc

```diff
@@ -27,3 +27,12 @@ New components are not automatically added to the `DataScienceCluster` object du
 * Notebook images are integrated into the image stream during the upgrade and subsequently appear in the {productname-short} dashboard. 
 +
 NOTE: Notebook images are constructed externally; they are prebuilt images that undergo quarterly changes and they do not change with every {productname-short} upgrade.
+
+ifdef::upstream[]
+[IMPORTANT]
+====
+After upgrading from {productname-short} 2.19 to 2.20, existing instances of the model registry component are not updated, which causes the instance pods to use older images than the ones referenced by the operator pod.
+
+To resolve this issue, you can create a new model registry instance that points to the same backing database, and delete the old model registry. Your new model registry instance will retain all existing registered models and their metadata. 
```

Comment:
rewording suggestion: 
- remove "you can"
- text changes/additions shown below in bold font
( is a "backing" database the same thing as a "backup" database?)

To resolve this issue, create a new model registry instance that points to the **existing backup** database, and **then** delete the old model registry **instance**. **The** new model registry instance **contains** all existing registered models and their metadata.

Reply:
Thanks Melissa! Couple of points -

- I hadn't used 'do x, and then do y' wording because the user can create/delete in any order, and might choose to delete first if they want to use the same model registry name for the new registry as they did for the original.
- 'Backing' database is just the database the model registry was configured with.

How about:

_To resolve this issue, create a new instance of each existing model registry, using the same database configuration, and delete the old model registry instance. The new model registry instance contains all existing registered models and their metadata._

Could also include this if it would be clearer RE: order:
_Model registry names must be unique. If you want to use the same model registry name as your original registry, you must delete the original registry before creating the new one._